feature-matcher-generator/src/main/java/ru/yandex/qatools/processors/matcher/gen/MatcherFactoryGenerator.java

```diff
@@ -62,7 +65,8 @@ public boolean process(Set<? extends TypeElement> annotations, RoundEnvironment
 
             VelocityEngine engine = engine();
 
-            FillMapWithFieldsProcess fillMapWithFields = fillMapOfClassDescriptionsProcess();
+            FillMapWithFieldsProcess fillMapWithFields =
```

Comment:
Yep, i know. yandex-qatools uses 120 symbols limit (we should write contrib.md somewhere i think)